src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/DetectPreviewFeatureUnitTests.cs

```diff
@@ -435,8 +424,7 @@ public async Task TestUnmarkedPreviewProperty()
             var csInput = @" 
         using System.Runtime.Versioning; using System;
         namespace Preview_Feature_Scratch
-        {" +
-@"
+        {
```

Comment:
Sigh, I'm hoping this is not another spot where the IOperation APIs are insufficient. If I have the following test case:
```csharp
            class Program : IProgram
            {
                static void Main(string[] args)
                {
                    Program program = new Program();
                    bool getter = program.[|AGetter|];
                }

                public bool AGetter
                {
                    [RequiresPreviewFeatures]
                    get
                    {
                        return true;
                    }
                }
            }
```

Currently we're registered to inspect OperationKind.PropertyReference, so we get a callback when `program.AGetter` is encountered. I inspect the operation and perform `(IPropertyReferenceOperation)operation.Property which returns the `AGetter` symbol. There's no way to know if the current operation is a getter or a setter it looks like? :/  Is that true @jmarolf @sharwell @mavasani ? I searched for something like OperationKind.GetProperty, but that doesn't exist, so I suspect the IOperation APIs don't contain specific information for getters/setters.

Reply:
Not sure what exactly you are asking for or trying to do, but this might what you are asking https://github.com/dotnet/roslyn-analyzers/blob/d0663cca11722045b9aeabb5d0d5335042084e13/src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzer.cs#L929

Reply:
Yup, this is what I was looking for! Thanks!!!!